cli/user/src/main/java/io/pravega/cli/user/config/InteractiveConfig.java

```diff
@@ -57,16 +57,26 @@ public class InteractiveConfig {
     private Level logLevel;
 
     public static InteractiveConfig getDefault() {
+        String controllerURI = "localhost:9090";
```

Comment:
the variables can be `private` if not being used anywhere else

Reply:
@shshashwat,  Access modifier can be used for local variables however I have not understand reason to define this variable inside method body, we can define as class level variable with default initialize, isn't @anju-c-das  ?

Reply:
No, a class level variable is not required in this case. It's being used just one time to update the value before populating the config.

Reply:
`String systemEnvURI = System.getenv("PRAVEGA_CONTROLLER_URI")`
`String controllerUri = systemEnvURI != null? systemEnvURI : "localhost:9090"`
You can use something like this also.